lib/mpl_toolkits/axisartist/grid_finder.py

```diff
@@ -121,21 +121,18 @@ def inverted(self):
 
 
 class GridFinder:
+    """
+    Internal helper for `.GridHelperCurveLinear`, with the same constructor parameters;
+    should not be directly instantiated.
+    """
+
     def __init__(self,
                  transform,
```

Comment:
Modify this to `aux_trans` for consistency (as this is the internal one, probably makes more sense to modify it here)?